### DISCUSSION TITLE
GiD version compatibility

### DISCUSSION THREAD

**Discussion by PetitPipi** [Anura3D/Anura3D_OpenSource#111] [Q&A] [answered]
Hi Team,
I was about to purchase a GiD license, but has some doubts on versions.  
The 2022 tutorial manual was using GiD version 14.0.1 as example.  I was wondering if higher version of the software(like GiD 16 or 15) works well with Anura3D? 
I  tested the GiD 15 and 16 by adding Anura3D problem type manually into the problemtype folder (in Win64 OS), and using a previous project file to regenerate mesh and Anura3D files. They looks fine, but I'm still not sure about it. 

I intend to buy a newer version if possible, so your comment about this issue could be very helpful to me. 
Thanks!

**Comment by ayerrocolom** [comment, marked as answer]
Hello,

Tomorrow we are releasing the Anura3D v2023, which has a new problemtype that works with GiD 16. If you want to know more about it, I recommend you attend the online Anura3D workshop tomorrow. Find the agenda and registration link here: [https://github.com/Anura3D/Anura3D_OpenSource/wiki/Anura3D-Online-Workshop-2023-&-official-release](url)

Best,
Alba

**Comment by RBH2020** [comment]
Hello,

I can not assign the material in GIDv16 ! is it normal? I can't carry on

**Reply by mikael-onsilva** [reply]
Hello, @RBH2020. I'm using v16 and works fine (at least for material assignment).

Are you facing an error or just can't figure it out how to do it?

Best regards,
Mikael

**Reply by RBH2020** [reply]
![ERREUR-GID-ANCIEN](https://github.com/Anura3D/Anura3D_OpenSource/assets/167431301/c861f9d8-a3a8-4a93-b2db-b599bf4a7aca)
Hello, 
Thank you for your reply.
Please find attached the error message that I receive after calculating 
the file .GOM always 0 ko 

Best regards, 
Rania BEN HMIDA

**Reply by mikael-onsilva** [reply]
It looks like your .GOM file is empty, or maybe not fully defined.

**Reply by RBH2020** [reply]
Hello, 

Yes always like that I tried so manay examples; D don't know from where come the problem.!

**Reply by mikael-onsilva** [reply]
Try to re-download and re-compile Anura3D and replace the GiD problemtype folder.

**Reply by mikael-onsilva** [reply]
Hi @acrodrigud. I didn't had this issue, actually. But if your .GOM file is empty, like @RBH2020, the issue is probably with the script in GiD to generate the Anura3D files. My first try would be re-download and re-compile both Anura3D and the problemtype and/or even GiD itself.

@RBH2020, did it worked for you?

**Reply by RBH2020** [reply]
Hello, 

I uninstalled twice the anura3D zip file, visul studio, intel base + intel HPC and i Compile and i did find any problems 
I put the file anura3D in Gid software, and the file anura3D.gid in problem type , i follow all the instruction in the different tutoriels. 

When i make the final step on a GiD which is the calculation "Generate anura3D files" this message appear 
invalid command name "customlib::SetBaseRoot"
invalid command name "::tkerror"
    while executing
"::tkerror $err"
then when i open the file "calculate.bat , i find this error also 
![error-calcul bate](https://github.com/Anura3D/Anura3D_OpenSource/assets/167431301/cac80bc7-8f52-4ffc-a00c-654fad02f083)

![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/167431301/7fb4b369-1d9c-442b-9a8b-b12d68aa7d5f)

Yours sincerely, 
RBH

**Reply by mikael-onsilva** [reply]
@RBH2020, try to run the analysis I've sent on my previous message in the WeTransfer link. If it works, the problem might be on your Anura3D, if it don't, it's probably something on your computer.

If it runs, I would suggest to replace the Anura3D.exe that comes into the "RainfallInfiltration.A3D" folder with the .exe you've compiled and then try to calculate. If it does not work, the problem is in your Anura3D compilation.

Let me know if it helps.

Best regards,
Mikael

**Reply by mikael-onsilva** [reply]
Just to make clear, this is the link (it's availiable just for 7 days): https://we.tl/t-XvZOEaX7p4

**Comment by mikael-onsilva** [comment]
I've tried to upload here, but the file is too big, so I've uploaded on WeTransfer both the problemtype folder and an example that runned for me.

I've created it on GiD 16.0.6.

This is the link (it's availiable just for 7 days): [https://we.tl/t-XvZOEaX7p4](https://we.tl/t-XvZOEaX7p4)
